### ISSUE TITLE
[resource][image] changed outside of Terraform

### ISSUE THREAD

**Issue by treywelsh** [OpenNebula/terraform-provider-opennebula#213]
Provider: 0.4.0
Terraform: from 0.15.4

After applying a VM disk resize a diff appear at plan in terraform:
```
Note: Objects have changed outside of Terraform
...
  # opennebula_image.test has been changed
  ~ resource "opennebula_image" "test-01" {
      ~ computed_size = 1024 -> 1152
        id            = "4067"
        name          = "test-01"
        # (13 unchanged attributes hidden)
    }
```
It's due to the fact that the resize of the disk also resize the image, so we have inter resource relationship.
The `computed_size` was introduced by issue #104  PR #124 in 0.4.0 release

**Comment by treywelsh** [comment]
The `computed_size` attribute was introduced to let the user know the image size on cloud side. The current image `size` attribute only allow the user to set a size manually and has `ForceNew` behavior so it should be handled with care.

Don't know how to fix this. I'm not even sure that I should consider this as a bug.
I don't know any tool in terraform SDK to manage inter ressource relationship.
Related terraform issue on expected `Objects have changed outside of Terraform`: https://github.com/hashicorp/terraform/issues/28803
At least to only update the state we could run a `terraform refresh`.

By the way I already tried to fix the same kind of diff in several places:
- #190 
- #163 
- #160 

These issues should be reviewed as well if we find a solutions to avoid this problem, just in case I tried to fix something that wasn't a real problem...
